src/tools/common/sss_tools.c

```diff
@@ -300,6 +263,49 @@ void sss_tool_usage(const char *tool_name, struct sss_route_cmd *commands)
     sss_tool_print_common_opts(min_len);
 }
 
+static int tool_cmd_init(struct sss_tool_ctx *tool_ctx,
+                         struct sss_route_cmd *command)
+{
+    int ret;
+
+    if (command->flags & SSS_TOOL_CMD_FLAG_NO_CONFDB) {
+        /* This tool does not need to connect to confdb.
```

Comment:
Sorry I should have notice it earlier, but this flag might be a little bit confusing.

The function `tool_cmd_init` does not initialize only confdb but also `tool_ctx->domains` and `tool_ctx->default_domain`. I know that we cannot initialize domains without confdb but
the name of flag should be crystal clear and should not any cause confusion.